meeteval/wer/__main__.py

```diff
@@ -189,174 +181,150 @@ def _save_results(
     }, per_reco_output)
 
     # Compute and save average
-    _average(per_reco.values(), average_output)
-
-
-def _print_human_readable(d):
-    from pprint import pprint
-    for k, v in d.items():
-        print(f'{k}: ', end='')
-        pprint(v)
+    average = combine_error_rates(*per_reco.values())
+    _dump(dataclasses.asdict(average), average_output)
 
 
-def _average(error_rates: List[ErrorRate], out: Path, print_human_readable=False):
-    average = combine_error_rates(*error_rates)
-    if print_human_readable:
-        _print_human_readable(dataclasses.asdict(average))
-    _dump(dataclasses.asdict(average), out)
-
-
-@click.group(help='MeetEval meeting evaluation tool')
-def cli():
-    pass
+parser = argparse.ArgumentParser(
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter
+)
+parser.add_argument('--version', action='store_true', help='Show version')
 
+commands = parser.add_subparsers(title='Subcommands')
 
-average_out_option = click.option(
-    '--average-out', default='{parent}/{stem}_{wer}.json',
+common_wer_parser = argparse.ArgumentParser(add_help=False)
+common_wer_parser.add_argument(
+    '-r', '--reference', help='Reference file in STM format',
+    nargs='+',
+    action='append',
+    # type=argparse.FileType('r'),
+)
+common_wer_parser.add_argument(
+    '-h', '--hypothesis', help='Hypothesis file in STM format',
+    # type=argparse.FileType('r'),
+    nargs='+',
+    action='append',
+)
+common_wer_parser.add_argument(
+    '--average-out',
+    default='{parent}/{stem}_{wer}.json',
```

Comment:
Would the default be set with `command_parser.set_defaults(func=fn)`, when the function defines a default? If yes, could you change it, that the function defines the default?